app/src/main/java/fr/gaulupeau/apps/Poche/data/dao/AnnotationDao.java

```diff
@@ -0,0 +1,243 @@
+package fr.gaulupeau.apps.Poche.data.dao;
+
+import java.util.List;
+import android.database.Cursor;
+import android.database.sqlite.SQLiteStatement;
+
+import org.greenrobot.greendao.AbstractDao;
+import org.greenrobot.greendao.Property;
+import org.greenrobot.greendao.internal.DaoConfig;
+import org.greenrobot.greendao.database.Database;
+import org.greenrobot.greendao.database.DatabaseStatement;
+import org.greenrobot.greendao.query.Query;
+import org.greenrobot.greendao.query.QueryBuilder;
+
+import fr.gaulupeau.apps.Poche.data.dao.entities.Annotation;
+
+// THIS CODE IS GENERATED BY greenDAO, DO NOT EDIT.
```

Comment:
the 8 occurances of this comment shall be removed according to https://github.com/wallabag/android-app/issues/1399#issuecomment-2127209508